docs/high-availability.md

```diff
@@ -1,22 +1,86 @@
 # Control Plane High Availability
 
-The configuration of a high availability control plane for k0s requires the deployment of both a load balancer and a cluster configuration file.
+You can create high availability for the control plane by distributing the control plane across multiple nodes and installing a load balancer on top. Etcd can be colocated with the controller nodes (default in k0s) to achieve highly available datastore at the same time.
+
+![k0s high availability](img/k0s_high_availability.png)
+
+## Network considerations
+
+You should plan to allocate the control plane nodes into different zones. This will avoid failures in case one zone fails.
+
+For etcd high availability it's recommended to configure 3 or 5 controller nodes.
+
+- With 3 nodes it's possible to lose one node and still have the quorum with 2 working nodes.
```

Comment:
maybe we can add a link to the official etcd faqs about this subject? https://etcd.io/docs/v3.3/faq/#why-an-odd-number-of-cluster-members